client/src/App.js

```diff
@@ -22,7 +24,8 @@ export default class App extends Component {
     return (
       <div>
         <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
+        <Route exact path="/" component={MovieList} />
+        <Route path="/movies/:id" render={(props) => <Movie {...props} addToSavedList={this.addToSavedList} />} />
```

Comment:
```suggestion
        <Route path="/movies/:id" 
            render={props => (
                <Movie {...props} addToSavedList={this.addToSavedList} />
            )} 
        />
```
when the line gets too long, always remembr to break the code down vertically. this makes your code easier to read for other team members. also, when passing only one parameter, you don't need to wrap it in paranthesis.